modules/apps/roles/roles-admin-test/src/testIntegration/java/com/liferay/roles/admin/internal/exportimport/data/handler/test/RoleStagedModelDataHandlerTest.java

```diff
@@ -60,66 +60,22 @@ public class RoleStagedModelDataHandlerTest
 	public static final AggregateTestRule aggregateTestRule =
 		new LiferayIntegrationTestRule();
 
+	@Override
 	@Test
-	public void testImportRoleWithUserGroup() throws Exception {
-		initExport();
-
-		UserGroup userGroup = _userGroupLocalService.addUserGroup(
-			StringPool.BLANK, TestPropsValues.getUserId(),
-			stagingGroup.getCompanyId(), RandomTestUtil.randomString(),
-			RandomTestUtil.randomString(),
-			ServiceContextTestUtil.getServiceContext(
-				stagingGroup.getGroupId()));
-
-		Role role = RoleTestUtil.addRole(RoleConstants.TYPE_REGULAR);
-
-		_groupLocalService.addRoleGroup(
-			role.getRoleId(), userGroup.getGroupId());
-
-		Assert.assertEquals(
-			1, _roleLocalService.getAssigneesTotal(role.getRoleId()));
-
-		StagedModelDataHandlerUtil.exportStagedModel(portletDataContext, role);
-
-		String originalName = PrincipalThreadLocal.getName();
-		PermissionChecker originalPermissionChecker =
+	public void testStagedModelDataHandler() throws Exception {
+		_company = CompanyTestUtil.addCompany();
+		_originalName = PrincipalThreadLocal.getName();
+		_originalPermissionChecker =
```

Comment:
sorry @wanderlast for this new change but I'm not sure about these rows because they are used only in the specific tests so why do we need to have them "gloabal"? Can we just add them where needed?

Reply:
We can just add them where needed. I've made the change.